### DISCUSSION TITLE
Please add FPS Option

### DISCUSSION THREAD

**Discussion by angyone1** [Drumber/RemoteLight#21] [Ideas]
Hello, I really like this program for openrgb, especially the effects! is it possible to add the option to increase or decrease the speed in frames per second of the displayed effects? there is a plugin for openrgb called Effects Engine Plugin (by herosilas12) which has this option in effects. Thank you very much and I apologize because I am totally noob and my English is awful ...

**Comment by Drumber** [comment]
Hi, if I understand correctly you want to control the speed in Frames per second and not as currently the delay in miliseconds. I could add a option for changing the speed slider mode. I've a bunch of other things to improve and add, but I will add it to my ToDo list ;)

**Reply by angyone1** [reply]
wow thank you so much! yes you got it right, in addition to the slider to change the speed of the animation I would like another slider to control the fluidity of the animation, more frames per second help a lot the eye not to get tired and it benefits all the harmony of the animation !  thanks so much !

**Reply by Drumber** [reply]
I almost forgot: if you want to send new data at a higher rate, you can set the "Output delay" option in the settings to a lower value (e.g. 10 ms). This will give you a higher update/frame rate.

**Reply by angyone1** [reply]
seems a little better... in settings I set to 5ms and some animation seems a little better. I dont know if this option are real fps option.. thanks
